README.md

```diff
@@ -1 +1,45 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## ABOUT ME
+
+![Kagwiria Nkonge.](https://github.com/Nkonge/hw01/blob/master/Images/Profile_pic.jpg)
+
+<center>
+
+My name is Nkonge Kagwiria Regina, a graduate student (class of 2020) at Harris School of Public Policy, *studying MA International Development and Policy. I come from Kenya and I am an Obama Foundation Scholar. My policy interests are international development, security, conflict and humanitarian intervention as well as technology and education*. 
+Prior to coming to Harris my work focused on technology for development particularly the application of technology in higher education, refugee camps and hard to reach areas. I hold an MSc. in Information Systems Security and Digital Forensics.
+**I am excited to be in this class!**
+
+</center>
+
+## PURPOSE OF THIS PROJECT
+<center>
+
+The goal of this project is to test your software installation, our GitHub setup, and our homework submission process via pull request, as well as demonstrate competency in Markdown.
+
+</center>
+
+## HOW I CREATED THIS FILE
+
+    1. I installed Git (R studio was already installed in my computer)
+    2. I forked and cloned the hw01 repository
+    3. I opened the README file using R Studio and Git as the version control software
+    4. I edited the README using RMarkdown: added the following sections - about me, purpose of this project, how I created this file
+    5. I staged and commited the file, then pushed it to GitHub
```

Comment:
Check the final version of the document on GitHub before you submit it. This list is formatted incorrectly so it does not render appropriately. You don't need to indent the elements, nor do you need to manually number them. Markdown will automatically increment the list items.

```
1. I installed Git (R studio was already installed in my computer)
1. I forked and cloned the hw01 repository
1. I opened the README file using R Studio and Git as the version control software
1. I edited the README using RMarkdown: added the following sections - about me, purpose of this project, how I created this file
1. I staged and commited the file, then pushed it to GitHub
```

1. I installed Git (R studio was already installed in my computer)
1. I forked and cloned the hw01 repository
1. I opened the README file using R Studio and Git as the version control software
1. I edited the README using RMarkdown: added the following sections - about me, purpose of this project, how I created this file
1. I staged and commited the file, then pushed it to GitHub